docs/components/form-associated.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Form-Associated Components
+sidebar_label: Form-Associated Components
+description: Form-Associated Stencil Components
+slug: /form-associated
+---
+
+# Building Form-Associated Components in Stencil
+
+As of $VERSION Stencil has experimental support for form-associated custom
+elements. This allows Stencil components to participate in a rich way in HTML
+forms, integrating with native browser features for validation and
+accessibility while maintaining encapsulation and control over their styling
+and presentation.
+
+:::caution
+Browser support for the APIs that this feature depends on is still relatively
+low ([~84% as of this writing](https://caniuse.com/?search=attachInternals))
+and the Stencil team has no plans at present support or incorporate any
+polyfills for the browser functionality. Accordingly, this is a 'use at your
+own risk' feature: it is up to you as a developer to ensure the browsers you
+need to support have shipped the necessary APIs.
```

Comment:
This reads a little too adversarial to me - I think we can soften this a little bit by removing the content that precedes the colon like so:
```suggestion
polyfills for the browser functionality. It is the responsibility of the developer to ensure the browsers you need to support have shipped the necessary APIs.
```

Reply:
I'm also a little concerned about this figure (~84% as of this writing) changing over time/us not keeping it up to date. What're you're thoughts on something like:

> Browser support for the APIs that this feature depends are still evolving.
At this time, the Stencil team has no plans at present support or incorporate any
polyfills for the browser functionality. It is the responsibility of the developer to ensure the browsers you need to support have shipped the necessary APIs.

Reply:
Sure, although I think "as of this writing" indicates it's not going to be current forever - once it's higher (above what threshold I don't know) I guess the note should just go away entirely